backend/src/authentication/auth.js

```diff
@@ -1,18 +1,18 @@
 const jwt = require('jsonwebtoken')
-const User = require('../models/user')
+const UserAccount = require('../models/userAccount')
 
 const auth = async (req, res, next) => {
     try {
         const token = req.header('Authorization').replace('Bearer ', '')
         const decoded = jwt.verify(token, process.env.JWT_SECRET)
-        const user = await User.findOne({ _id: decoded._id, 'tokens.token': token })
+        const userAccount = await UserAccount.findOne({ _id: decoded._id, 'tokens.token': token })
 
-        if (!user) {
+        if (!userAccount) {
             throw new Error()
         }
 
         req.token = token
-        req.user = user
+        req.userAccount = userAccount
```

Comment:
same thing here, let this be req.user coz otherwise there's a lot of refactoring to be done